packages/sdk/src/get-generators/utils/check-feature-flags/checkFeatureFlags.ts

```diff
@@ -1,25 +1,12 @@
 import type { ConfigMetaFormat } from '../../../engine-commands'
 import { forbiddenTransactionsWithProxyFlagMessage } from './forbiddenTransactionsWithProxyFlagMessage'
-import { mongoFeatureFlagMissingMessage } from './mongoFeatureFlagMissingMessage'
 import { proxyFeatureFlagMissingMessage } from './proxyFeatureFlagMissingMessage'
 
 export function checkFeatureFlags(config: ConfigMetaFormat) {
-  checkMongoFeatureFlag(config)
   checkProxyFeatureFlag(config)
   checkForbiddenTransactionsWithProxyFlag(config)
 }
 
-function checkMongoFeatureFlag(config: ConfigMetaFormat) {
-  if (
-    config.datasources.some((d) => d.provider === 'mongodb') &&
-    !config.generators.some((g) => {
-      return g.previewFeatures.some((previewFeature) => previewFeature.toLowerCase() === 'mongoDb'.toLowerCase())
-    })
-  ) {
-    throw new Error(mongoFeatureFlagMissingMessage)
-  }
-}
```

Comment:
Same comment as everywhere, might be useful to keep around as the functionality is generally good.
(Might be generalized though - so maybe just move to an issue to generalize instead of hardcoding `mongodb`)